src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_vload8.cpp

```diff
@@ -1,147 +0,0 @@
-﻿// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include "activation/activation_kernel_base.h"
-#include "eltwise_kernel_vload8.h"
-#include "kernel_selector_utils.h"
-#include <string>
-#include <algorithm>
-
-namespace kernel_selector {
-
-ParamsKey EltwiseKernel_vload8::GetSupportedKey() const {
-    ParamsKey k;
-    k.EnableInputDataType(Datatype::F16);
-    k.EnableInputDataType(Datatype::F32);
-    k.EnableOutputDataType(Datatype::F16);
-    k.EnableOutputDataType(Datatype::F32);
-    k.EnableAllInputLayout();
```

Comment:
Yes, that's right. I missed the format you mentioned. I also found this problem while checking gen9 perf. It happens on model person-detection-0301.xml.
Thank you. This will be fixed.